### PR TITLE
enable merge button for poi-to-polygon reviews

### PR DESCRIPTION
**Pull request by brianhatchl**
refs #644

### CODE REVIEW

**Comment by brianhatchl** [comment]
jenkins, please run cucumber

**Comment by brianhatchl** [comment]
jenkins, please run cucumber

File: js/hoot/control/conflicts/map/FeatureHighlighter.js [hunk 1]
```diff
@@ -35,8 +35,14 @@ Hoot.control.conflicts.map.featurehighlighter = function (context)
...
-        if ((feature && againstFeature) && (feature.id.charAt(0) === 'n' && againstFeature.id.charAt(0) === 'n')) {
+
+        var currentReviewable = _parent().actions.traversereview.getCurrentReviewable();
+        var relId = 'r' + currentReviewable.relationId + '_' + currentReviewable.mapId;
+        var relation = context.entity(relId);
+
+        //Both POI-to-POI and POI-to-Polygon review features
+        //are poi reviews that can be merged.
+        if ((relation.tags['hoot:review:type'] === 'POI to Polygon') || ((feature && againstFeature) && (feature.id.charAt(0) === 'n' && againstFeature.id.charAt(0) === 'n'))) {
             //Show merge button
```
Comment (sisskind) [inline js/hoot/control/conflicts/map/FeatureHighlighter.js]: What if the relation.tags has no 'hoot:review:type' property?

Comment (brianhatchl) [inline js/hoot/control/conflicts/map/FeatureHighlighter.js]: It should just evaluate to undefined and then the conditional will be false.

**Pull request merged**